.github/ISSUE_TEMPLATE/chart_upgrade.md

```diff
@@ -7,20 +7,20 @@ assignees: ''
 
 ---
 
-**What Chart should be upgraded?**
+**What Chart should be upgraded?**:
```

Comment:
```suggestion
**What Chart should be upgraded?**
```

Reply:
I can't remove it else it doesn't comply with MD036: Emphasis used instead of a header.
So I think this is a good enough circumvention here.

I'll see if I can do some other type of rework when I get around to update the templates.